app/helpers/date-format.js

```diff
@@ -0,0 +1,12 @@
+import { helper } from '@ember/component/helper';
+import { format } from 'date-fns';
+import { dateHelper } from 'frontend-kaleidos/utils/date-helper';
+
+export default helper(function dateFormat([dateOrString, formatString]) {
+  if (formatString) {
+    const dateObject = dateHelper(dateOrString);
+    if (dateObject) {
+      return format(dateObject, formatString);
+    }
+  }
```

Comment:
It's maybe a bit more clean (and less code duplication) if we extract this snippet into a util instead of only the `dateHelper`. As such it can be imported in the other helpers and used like
```javascript
export default helper(function datePhrase([dateOrString]) {
  return dateFormat(dateOrString, 'd MMMM yyyy');
});
```